src/theme/material/dialog.m.css

```diff
@@ -1,2 +1,50 @@
+/* The root class for the Dialog */
 .root {
+	composes: mdc-dialog from '@material/dialog/dist/mdc.dialog.css';
+}
+
+/* Added to `.root` for an open dialog */
+.open {
+	composes: mdc-dialog--open from '@material/dialog/dist/mdc.dialog.css';
+}
+
+/* Contains the dialog */
+.main {
+	composes: mdc-dialog__container, mdc-dialog__surface from '@material/dialog/dist/mdc.dialog.css';
+	outline: 0;
+}
+
+/* Contains the title of the dialog */
+.title {
+	composes: mdc-dialog__title from '@material/dialog/dist/mdc.dialog.css';
+	padding-top: 20px;
+}
+
+.title:before {
+	display: none;
+}
+
+/* Contains the content of the dialog */
+.content {
+	composes: mdc-dialog__content from '@material/dialog/dist/mdc.dialog.css';
+}
+
+.close {
+	background: none;
+	border: none;
+	cursor: pointer;
+	font-size: 0;
+	padding: 8px;
+	position: absolute;
+	right: 16px;
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.close i {
```

Comment:
I think this selector will cause a warning in the build

Reply:
You're right. I copied that from the Dojo theme, so I've fixed it as well.